holidays/countries/cook_islands.py

```
@@ -0,0 +1,148 @@
+#  holidays
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)